### PR TITLE
Update Observability Guide

### PR DESCRIPTION
**Pull request by Anoukh**

### CODE REVIEW

File: website/two-column-pages/docs/learn/how-to-observe-ballerina-code.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ make tracing and metrics monitoring more informative.
 ## Getting Started
 This section focuses on enabling Ballerina service observability with some its default supported systems.
 [Prometheus] and [Grafana] are used for metrics monitoring, and [Jaeger] is used for distributed tracing. 
-Ballerina logs can be fed to any external log monitoring system like Elastic Stack to perform log monitoring and 
+Ballerina logs can be fed to any external log monitoring system like [Elastic Stack] to perform log monitoring and
```
Comment (samgnaniah) [inline website/two-column-pages/docs/learn/how-to-observe-ballerina-code.md]: Why have we added brackets?

File: website/two-column-pages/docs/learn/how-to-observe-ballerina-code.md [hunk 2]
```diff
@@ -418,32 +418,31 @@ $ tail -f ~/wso2-ballerina/workspace/ballerina.log
...
-2. Logstash - Used to process and structure the log files received from Filebeat and send Elasticsearch.
+2. Logstash - Used to process and structure the log files received from Filebeat and send to Elasticsearch.
 3. Elasticsearch - Storage and indexing of the logs received by Logstash.
 4. Kibana - Visualizes the data stored in Elasticsearch
 
-Elasticsearch and Kibana are provided as cloud services from https://www.elastic.co/cloud with a trial period.
-We only have to set up Logstash and Filebeat containers on-premise if you are going ahead with cloud services.
-If you are not opting for the cloud service, you can use Docker containers for all the tools.
+Elasticsearch and Kibana are provided as [Cloud Services](https://www.elastic.co/cloud)
+Alternatively, docker containers can be used to set up Elasticsearch and Kibana as well.
```
Comment (samgnaniah) [inline website/two-column-pages/docs/learn/how-to-observe-ballerina-code.md]: Change 'docker' to 'Docker'. Apply to any other instances of this,

File: website/two-column-pages/docs/learn/how-to-observe-ballerina-code.md [hunk 3]
```diff
@@ -418,32 +418,31 @@ $ tail -f ~/wso2-ballerina/workspace/ballerina.log
...
 # Filebeat Image
-$ docker pull docker.elastic.co/beats/filebeat:6.2.2
+$ docker pull docker.elastic.co/beats/filebeat:6.2.4
 # Logstash Image
-$ docker pull docker.elastic.co/logstash/logstash:6.2.2
+$ docker pull docker.elastic.co/logstash/logstash:6.2.4
 ```
 
-**Step 2:** Start elastic search and Kibana containers with below commands.
+**Step 2:** Start Elasticsearch and Kibana containers by executing below commands.
```
Comment (samgnaniah) [inline website/two-column-pages/docs/learn/how-to-observe-ballerina-code.md]: Let's say 'the following commands.'

File: website/two-column-pages/docs/learn/how-to-observe-ballerina-code.md [hunk 4]
```diff
@@ -453,17 +452,7 @@ Execute the following command to do that.
...
-* `--link` flag is used to connect Docker container to another container. The container can consume services using the
-hostname specified.
-
-The Kibana container links to the Elasticsearch container and can use the `elasticsearch` hostname to talk to that
-container.
-
-**Step 3:** Next step is, configuring Logstash to format the Ballerina logs. In order to do this, you need to create a file named `logstash.conf` with the following content.
-
-For this example, let's save this file at `/tmp/pipeline/logstash.conf`. This should be taken note of because, when starting the Logstash container, this file should be bind-mounted onto the container. Logstash container is configured to read a configuration file named `logstash.conf` at startup.
+**Step 3:** Create a `logstash.conf` file in `/tmp/pipeline/` directory and include following content in the file.
```
Comment (samgnaniah) [inline website/two-column-pages/docs/learn/how-to-observe-ballerina-code.md]: 'Change to 'Create a `logstash.conf` file in the `/tmp/pipeline/` directory and include the following content in the file.'

File: website/two-column-pages/docs/learn/how-to-observe-ballerina-code.md [hunk 5]
```diff
@@ -489,18 +478,13 @@ Here the 3 stages are specified in the pipeline. Input is specified as beats and
...
-
 **Step 4:** Start the Logstash container by the following command.
 
 ```bash
-$ docker run -h logstash --name logstash --link elasticsearch:elasticsearch -it --rm -v /tmp/pipeline:/usr/share/logstash/pipeline/ -p 5044:5044 docker.elastic.co/logstash/logstash:6.2.2
+$ docker run -h logstash --name logstash --link elasticsearch:elasticsearch -it --rm -v /tmp/pipeline:/usr/share/logstash/pipeline/ -p 5044:5044 docker.elastic.co/logstash/logstash:6.2.4
 ```
 
-**Step 5:** Configure Filebeat to ship the Ballerina logs. For this, you need to create a file named filebeat.yml with the following content. As an example, let's save this file at `/tmp/filebeat.yml`.
+**Step 5:** Configure Filebeat to ship the Ballerina logs. Create a `filebeat.yml` file in `/tmp/` directory and include following content in the file.
```
Comment (samgnaniah) [inline website/two-column-pages/docs/learn/how-to-observe-ballerina-code.md]: Change to 'Create a `filebeat.yml` file in the `/tmp/` directory and include the following content in the file.'

File: website/two-column-pages/docs/learn/how-to-observe-ballerina-code.md (hunk 1 above)
Comment (Anoukh) [reply website/two-column-pages/docs/learn/how-to-observe-ballerina-code.md]: It is a link. The link is given at the end of the file. Similar to line 16

**Pull request merged**